test/helpers/fixture_deploy_helper.rb

```diff
@@ -58,9 +58,9 @@ def deploy_global_fixtures(set, subset: nil, namespaced: true, **args)
     success
   end
 
-  def deploy_global_crd_fixtures(subset:, prune: true, clean_up: true, &block)
-    deploy_global_fixtures("crd", subset: subset, selector: 'app=krane', clean_up: clean_up, prune: prune,
-                                  namespaced: false, &block)
+  def deploy_global_fixtures_non_namespaced(fixture, subset:, prune: true, clean_up: true, &block)
```

Comment:
We talked IRL, might be nice to cut an issue to refactor this and make `deploy_global_fixtures` more flexible but outside the scope of this PR. cc @dirceu 👍